package.json

```diff
@@ -2,38 +2,51 @@
   "name": "molotov",
   "description": "Molotov: (Mix/Plug)-in loader for explosive composeability",
   "main": "molotov.js",
+  "version": "0.0.0-development",
   "directories": {
     "example": "examples"
   },
   "dependencies": {
-    "callsite": "^1.0.0",
-    "config": "^1.25.1",
+    "ajv": "^6.2.0",
     "fs-extra": "^4.0.1",
     "lodash": "^4.17.4",
-    "require-directory": "^2.1.1",
-    "try-require": "^1.2.1"
+    "require-directory": "^2.1.1"
   },
   "devDependencies": {
-    "ava": "^0.25.0",
+    "@commitlint/config-angular": "^6.1.2",
+    "babel-eslint": "^8.2.2",
+    "commitlint": "^6.1.2",
     "coveralls": "^3.0.0",
-    "eslint": "^3.16.0",
-    "eslint-config-airbnb-base": "^11.1.0",
-    "eslint-find-rules": "^1.13.0",
-    "eslint-plugin-ava": "^4.2.0",
-    "eslint-plugin-import": "^2.0.1",
+    "eslint": "^4.18.1",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-flowtype": "^2.46.1",
+    "eslint-plugin-import": "^2.9.0",
+    "flow-bin": "^0.66.0",
+    "flow-remove-types": "^1.2.3",
+    "flow-typed": "^2.3.0",
     "husky": "^0.14.0",
-    "npm-run-all": "^4.0.1",
-    "nyc": "^11.0.3",
+    "jest": "^22.4.2",
+    "jest-environment-node-debug": "^2.0.0",
     "semantic-release": "^6.3.2",
     "semantic-release-conventional-commits": "^1.0.2"
   },
   "scripts": {
-    "lint": "eslint .",
-    "test": "nyc ava && nyc report --reporter=html --reporter=lcov",
-    "precommit": "npm run lint",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "test": "npm run build && jest --coverage",
+    "test-ci": "npm run build && jest --coverage --runInBand",
+    "debug-test": "node --inspect --debug-brk ./node_modules/.bin/jest --env jest-environment-node-debug",
+    "precommit": "npm run build && npm run flow && npm run lint",
     "prepush": "npm test",
-    "coverage": "nyc --reporter=lcov --reporter=html check-coverage --statements 100 --lines 100 --branches 100 --functions 100",
-    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "prepublish": "npm run build",
```

Comment:
If you're running publish on node 8 you _might_ want to make this `prepack`, but either way 👍

Reply:
I think that's a good idea. Might need to make sure the .travis.yml is up to speed 👍